covid_api/db/utils.py

```diff
@@ -2,141 +2,86 @@
 
 import csv
 import json
-import re
 from datetime import datetime
-from typing import Dict, List, Optional, Set
+from typing import Dict, List
 
 import boto3
+from botocore import config
 
-from covid_api.core.config import DT_FORMAT, INDICATOR_BUCKET, MT_FORMAT
+from covid_api.core.config import DT_FORMAT, INDICATOR_BUCKET
 from covid_api.models.static import IndicatorObservation
 
 s3 = boto3.client("s3")
 
+_lambda = boto3.client(
+    "lambda",
+    region_name="us-east-1",
+    config=config.Config(
+        read_timeout=900, connect_timeout=900, retries={"max_attempts": 0}
+    ),
+)
 
-def gather_s3_keys(
```

Comment:
Gathering S3 keys, filtering etc, is no longer being performed by the API, but rather by the dataset metadata generator lambda function that runs daily